medusa/providers/torrent/html/archetorrent.py

```diff
@@ -0,0 +1,215 @@
+                return items
+
+            # Catégorie, Release, Date, DL, Size, C, S, L
+            labels = [label.get_text(strip=True) for label in torrent_rows[0]('th')]
+
+            for torrent in torrent_rows[1:]:
+                cells = torrent('td')
+                if len(cells) < len(labels):
+                    continue
+
+                try:
+                    torrent_id = re.search('id=([0-9]+)', cells[labels.index('Nom')].find('a')['href']).group(1)
+                    title = cells[labels.index('Nom')].get_text(strip=True)
+                    download_url = urljoin(self.urls['download'], '?id={0}&name={1}'.format(torrent_id, title))
+                    if not all([title, download_url]):
+                        continue
+
+                    # Chop off tracker/channel prefix or we cannot parse the result!
+                    if mode != 'RSS' and keywords:
+                        show_name_first_word = re.search(r'^[^ .]+', keywords).group()
+                        if not title.startswith(show_name_first_word):
+                            title = re.sub(r'.*(' + show_name_first_word + '.*)', r'\1', title)
+
+                    seeders = try_int(cells[labels.index('S')].get_text(strip=True))
+                    leechers = try_int(cells[labels.index('L')].get_text(strip=True))
+
+                    # Filter unseeded torrent
+                    if seeders < min(self.minseed, 1):
+                        if mode != 'RSS':
+                            log.debug("Discarding torrent because it doesn't meet the"
+                                      " minimum seeders: {0}. Seeders: {1}",
+                                      title, seeders)
+                        continue
+
+                    size_index = labels.index('Size') if 'Size' in labels else labels.index('Taille')
+                    torrent_size = cells[size_index].get_text()
+                    size = convert_size(torrent_size, units=units) or -1
+
+                    pubdate_raw = BS4Parser(torrent('a')[1]['onmouseover'][16:-2],
+                                            'html5lib').soup('div')[0].contents[1]
+                    pubdate = self.parse_pubdate(pubdate_raw, dayfirst=True)
+
+                    item = {
+                        'title': title,
+                        'link': download_url,
+                        'size': size,
+                        'seeders': seeders,
+                        'leechers': leechers,
+                        'pubdate': pubdate
+                    }
+
+                    if mode != 'RSS':
+                        log.debug('Found result: {0} with {1} seeders and {2} leechers',
+                                  title, seeders, leechers)
+
+                    items.append(item)
+                except (AttributeError, TypeError, KeyError, ValueError, IndexError):
+                    log.error('Failed parsing provider. Traceback: {0!r}',
```

Comment:
Can you replace this by: `log.exception('Failed parsing provider.')`. Dont forget to remove the traceback import.